src/main/java/org/jitsi/videobridge/simulcast/SimulcastReceiver.java

```diff
@@ -571,9 +630,11 @@ private void maybeTimeout(
             SimulcastStream cause,
             RawPacket pkt,
             SimulcastStream effect,
+            List<SimulcastStream> localSimulcastStreamFrameHistory,
```

Comment:
I think this (and endIndexInSimulcastStreamFrameHistory, too) needs a javadoc. It is not obvious what it is.